content/data/scripts/ctrlr_multi_common.lua

```diff
@@ -1354,7 +1354,7 @@ engine.addHook("On Frame", function()
 	if ScpuiSystem.data.memory.multiplayer_general.RunNetwork then
 		ScpuiSystem.data.memory.multiplayer_general.Context:updateLists()
 	end
-end, {}, function()
+end, {State="GS_STATE_BRIEFING"}, function()
```

Comment:
Ah this one is incorrect but I think you were led astray by the copy/pasted comment. This hook should run every frame in any multiplayer-related game state that needs to run network functions. The full list of supported states starts at line 14 of this same file.

If you wanted to change the comment while you're there it would be `--- During multiplayer run the network functions every frame`.

Reply:
Thanks, just updated. Happy to use a different strategy too, just let me know!